src/main/java/duke/Todo.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * Represents the todo tasks input by the user
+ */
+public class Todo extends Task {
+
+    /**
+     * Super constructor to the task class
+     *
+     * @param description the description of the todo task
+     * @return
+     * @throws
+     */
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Returns todo task in string format
+     *
+     * @param
+     * @return todo task in a string format
+     * @throws
+     */
+    public String getDescription() {
+        String newReply = super.description.replace("todo ", "");
+        return "[T]" + "[" + super.getStatusIcon() + "] " + newReply;
+    }
```

Comment:
You should also consider overwriting the toString() method of the task instead of creating a sepearte getDescription() method as the toString() method is not being used for anything else so you might as well repurpose it. This is will also help with printing as you do:
```Java 
System.out.println(event) 
``` 
instead of printing out the hashcode for the object its description will appear.